ethcore/src/client/client.rs

```diff
@@ -899,6 +900,7 @@ impl BlockChainClient for Client {
 		}
 		let options = TransactOptions { tracing: true, vm_tracing: false, check_nonce: false };
 		let mut tx = t.clone();
+		tx.gas_price = 0.into();
```

Comment:
That kind of breaks the whole purpose of accepting (optional) `gas_price` in the RPC, shouldn't the caller decide this?